src/App.tsx

```diff
@@ -16,35 +18,94 @@
 ];
 
 export const App: React.FC = () => {
+  let goods = [...goodsFromServer];
+  let isReverseCLicked = false;
```

Comment:
[CODE KNOWLEDGE] - Don't interact with DOM directly, use React as much as possible. `isReverseClicked` should be a state variable, not a regular variable. Otherwise, its changes won't trigger a re-render of the component.